UtopiaCity/UtopiaCity.csproj

```diff
@@ -45,8 +45,17 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Folder Include="Data\Migrations\" />
```

Comment:
seems like it is needed still

Reply:
I deleted and re-restored the database before the pull request, everything seemed to work fine. And I have a question how does it work if the folder with migrations is not included:)